site/content/en/docs/Guides/feature-stages.md

```diff
@@ -24,7 +24,6 @@ that can be found in the [Helm configuration]({{< ref "/docs/Installation/Instal
 
 The current set of `alpha` and `beta` feature gates are:
 
-{{% feature expiryVersion="1.25.0" %}}
```

Comment:
Shouldn't this entire block be deleted, not just the top and bottom feature codes?